.buildkite/gen-pipeline.sh

```diff
@@ -438,14 +438,14 @@ for test in ${tests[@]-}; do
   if [[ ${test} == *-gpu-* ]] || [[ ${test} == *-mixed-* ]]; then
     # if gloo is specified, run gloo gpu integration tests
     if [[ ${test} == *-gloo* ]]; then
-      run_gloo_integration ${test} "2x-gpu-g4"
+      run_gloo_integration ${test} "2x-gpu-v510"
```

Comment:
What is the format here? 2x means 2 GPUs? g4 or v510 is different GPU card?

Reply:
It's arbitrary.  Before, it was <# gpus per container>-<cpu or gpu>-<sku>.  Now, since we always use g4 for the sku, I replaced it with the version of the Buildkite stack.